src/main/scala/cromwell/authentication/DockerCredentials.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.authentication
```

Comment:
The `cromwell.authentication` package seems a little thin to warrant its own entire top-level package, in my opinion.  I'm not gonna fuss about it if you disagree though